test/Test/Pos/Ssc/GodTossing/ComputeSharesSpec.hs

```diff
@@ -37,22 +38,25 @@ spec = describe "computeSharesDistr" $ do
     validRichmenStakeWorksDesc = "Given a valid distribution of stake, calculating the\
     \ distribution of shares successfully works."
 
+computeShares' :: RichmenStake -> Either T.TossVerFailure T.SharesDistribution
+computeShares' stake = runExcept $ T.computeSharesDistrPure stake genesisMpcThd
```

Comment:
What? It seems useless, because `Either Patak` is instance of `MonadError Patak`.